src/components/Comments.jsx

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useState } from "react";
+import { getCommentsByArticleId } from "../api";
+
+function ShowComments({ article_id }) {
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    getCommentsByArticleId(article_id)
+      .then((response) => {
+        setComments(response);
+        setIsLoading(false);
+      })
+      .catch((error) => {
+        console.error("err computer says no.. comments here", error);
```

Comment:
Rather than console logging this, it would be great if there was some error state instead 👍